<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Numer drogi</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>np. G123111R</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Długość</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Kategoria</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>krajowa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wojewódzka</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>powiatowa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gminna</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wewnętrzna</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>inna</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Uwagi</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="maximum">
          <double>999999999.990000009536743</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Wartość drogi</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_auto">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>jednostek</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="save_btn">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="reject_btn">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit_auto</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>save_btn</tabstop>
  <tabstop>reject_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>save_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reject_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
